project/Settings.scala

```diff
@@ -13,14 +13,8 @@ import sbtdocker.{DockerPlugin, Instruction, Instructions}
 
 object Settings {
 
-  /* The reason why -Xmax-classfile-name is set is because this will fail
-     to build on Docker otherwise.  The reason why it's 200 is because it
-     fails if the value is too close to 256 (even 254 fails).  For more info:
-
-     https://github.com/sbt/sbt-assembly/issues/69
-     https://github.com/scala/pickling/issues/10
-
-     Other fancy flags from https://tpolecat.github.io/2017/04/25/scalac-flags.html.
```

Comment:
this flag is gone, 2.13 is aware of the issue and chooses an even more conservative default